docs/source/autograd.rst

```diff
@@ -11,6 +11,37 @@ Automatic differentiation package - torch.autograd
 
 .. autofunction:: grad
 
+.. _functional-api:
+
+Functional higher level API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    This API is experimental.
+
+This section contains the higher level API for the autograd that builds on the basic API above
+and allows you to compute jacobians, hessians, etc.
+
+This API works with user-provided functions that take only Tensors as input and return
+only Tensors.
+If your function takes other arguments that are not Tensors or Tensors for which you don't require gradients,
```

Comment:
nit: I'd be explicit here, "for which you don't require gradients" you mean -- the Tensors don't have `requires_grad` set, right?  i.e. I might have requires_grad set but not actually require the gradients.

Reply:
Is this better:
"If your function takes other arguments that should be ignored when computing the given quantity (for example inputs that are not Tensors, or Tensors that should be ignored when computing the quantity),"

The example below for the jacobian should give a concrete example.

Reply:
why not just "or Tensors that don't have requires_grad set" ?